numpy/testing/utils.py

```diff
@@ -903,12 +905,12 @@ def compare(x, y):
         # casting of x later.
         dtype = result_type(y, 1.)
         y = array(y, dtype=dtype, copy=False, subok=True)
-        z = abs(x-y)
+        z = abs(x - y)
 
         if not issubdtype(z.dtype, number):
             z = z.astype(float_)  # handle object arrays
 
-        return around(z, decimal) <= 10.0**(-decimal)
+        return z < 10**(-decimal)
```

Comment:
Only after a deprecation period, and of course it would get fixed before the deprecation could land anyway...